.github/CODEOWNERS

```diff
@@ -230,7 +230,7 @@
 /sdk/cosmos/                                                         @annatisch @Azure/azure-cosmos-python-sdk
 
 # ServiceLabel: %Cosmos %Service Attention
-#/<NotInRepo>/                                                       @pjohari-ms @simorenoh @AbhinavTrips @bambriz @pilchie
+#/<NotInRepo>/                                                       @pjohari-ms @simorenoh @AbhinavTrips @pilchie
```

Comment:
You can use ServiceOwners instead: e.g. https://github.com/Azure/azure-sdk-for-python/edit/main/.github/CODEOWNERS#L47